addons/hr_holidays/models/hr_leave.py

```diff
@@ -802,43 +758,11 @@ def add_follower(self, employee_id):
 
     @api.constrains('holiday_allocation_id')
     def _check_allocation_id(self):
-        for leave in self:
-            if leave.holiday_type == 'employee' and not leave.multi_employee and\
-                leave.holiday_status_id.requires_allocation == 'yes' and not leave.holiday_allocation_id:
-                raise ValidationError(_(
-                    'Could not find an allocation of type %(leave_type)s for the requested time period.',
-                    leave_type=leave.holiday_status_id.display_name,
-                ) + '\n- %s' % (leave.employee_id.name))
+        return
```

Comment:
Would be cool to add a comment explaining this method is deprecated and should be removed in master for all those emptied methods. Because if you blame the method, you won't fall on this commit.